WIP: Trimmmmmmmmming (#443)
AlexsLemonade/OpenPBTA-manuscript: content/05.limitations.md

## Limitations of Study

This study has some potential limitations.
The brain tumor samples were collected over decades and RNA samples were prepared using more than one type of library method (stranded or poly-A, **Figure {@fig:S7}A**) by multiple sequencing centers.
While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and possible sequencing center batch effect (**Figure {@fig:S7}C**), we also found a very unbalanced distribution of cancer groups by library preparation type (**Figure {@fig:S7}A**).

> We did not perform batch correction because removal of batch effects across unbalanced groups may induce false differences between groups in some cases [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012]. 

Reviewer: ```suggestion
We did not perform batch correction because removal of batch effects across unbalanced groups may induce false differences among groups [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012]. 
```

> Instead, we display only stranded RNA-Seq expression data, which comprised the vast majority of the PBTA cohort, for transcriptomic analyses presented in **Figure {@fig:4}** and **Figure {@fig:5}** to circumvent the batch differences.

Reviewer: ```suggestion
Instead, we analyzed only stranded RNA-Seq expression data, which comprised the vast majority of the PBTA cohort, for transcriptomic analyses presented in **Figure {@fig:4}** and **Figure {@fig:5}** to circumvent the batch differences.
```

> As batch correction strategy is highly dependent on an analyst’s goals [@doi:10.1016/j.tibtech.2017.02.012], we made expression matrices available by library type in the OpenPBTA data release for other researchers to employ based on their given analysis needs. 

Reviewer: something is funky with that literal quote..replacing it with a straight quote

```suggestion
As batch correction strategy is highly dependent on an analyst's goals [@doi:10.1016/j.tibtech.2017.02.012], we made expression matrices available by library type in the OpenPBTA data release for other researchers to employ based on their given analysis needs. 
```

A second potential limitation is that performing analyses with all samples, rather than samples with high tumor purity, might result in loss of information, such as subclonal variants or low-level oncogenic pathway expression.

> To confirm our analyses support true tumor biology, we performed the same transcriptomic analyses using only samples at or above median tumor purity within their cancer group (**Figure {@fig:S7}D-I**) and indeed, results were broadly consistent with those utilizing the full cohort of samples.

Reviewer: ```suggestion
To confirm our analyses support true tumor biology, we performed the same transcriptomic analyses using only samples at or above median tumor purity within their cancer group (**Figure {@fig:S7}D-I**), and indeed, results were broadly consistent with those utilizing the full cohort of samples.
```

Author: you sure like your commas ;)
